src/nni_manager/training_service/kubernetes/frameworkcontroller/frameworkcontrollerTrainingService.ts

```diff
@@ -266,24 +297,29 @@ class FrameworkControllerTrainingService extends KubernetesTrainingService imple
             throw new Error('frameworkcontroller Cluster config is not initialized');
         }
 
-        if(!this.frameworkcontrollerTrialConfig) {
+        if(!this.fcTrialConfig) {
             throw new Error('frameworkcontroller trial config is not initialized');
         }
         
         let taskRoles = [];
-        for(let index in this.frameworkcontrollerTrialConfig.taskRoles) {
+        for(let index in this.fcTrialConfig.taskRoles) {
```

Comment:
The same concern as above for "for...in..."

Reply:
Use 'for ... in ... ' in this place is to get the index of the array, and the index will be used in other purpose.